### PR TITLE
Update ADOPTERS.md

### PR DESCRIPTION
**Pull request by zxb789**
add use case as pr

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -5,4 +5,5 @@ This is a likely incomplete list of KubeVirt adopters - end-users and distributo
 | User | Since | Use-Case |
 |-|-|:-|
 | Red Hat | 2016 | As a distributor we are building OpenShift Virtualization on KubeVirt in order to enable VM workloads and -flows on Kubernetes. |
-| [Kubermatic](https://kubermatic.com) | 2019 | As a distributor we are running KubeVirt to enable VM workload on Kubermatic Virtualization. Follow the [link](https://www.kubermatic.com/products/kubevirt/) for more information. |
\ No newline at end of file
+| [Kubermatic](https://kubermatic.com) | 2019 | As a distributor we are running KubeVirt to enable VM workload on Kubermatic Virtualization. Follow the [link](https://www.kubermatic.com/products/kubevirt/) for more information. |
+| H3C | 2019 | As a distributor we are building KubeVirt in order to enable VM workloads on Kubernetes. |
```
Comment (dankenigsberg) [inline ADOPTERS.md:9]: @zxb789 would you care to elaborate about your use case a bit more?

Red Hat and Kubermatic are distributing KubeVirt to third parties. Is h3c a distributor as well? If so, can you share a URL for how can one download it?

BTW do you know why you show up twice on [cncf|https://github.com/cncf/landscape/blob/master/landscape.yml] ?

**Review by dankenigsberg — CHANGES_REQUESTED** [review]
would you modify the title of the pr to something like "add H3C to the list of ADOPTERS", and elaborate a bit more about what you are doing with KubeVirt?

**Comment by dankenigsberg** [comment]
@zxb789 you don't need to close the PR. It is better to keep it open and update it until we achieve a clear outcome.

**Comment by zxb789** [comment]
ok,3ks ,i forget add signoff tags; and i reopen pr https://github.com/kubevirt/kubevirt/pull/4714

**Comment by zxb789** [comment]
@dankenigsberg

File: ADOPTERS.md (hunk 1 above)
Comment (zxb789) [reply ADOPTERS.md]: our product name "cloudos " focus on private cloud ;and distributing to some company who need it,so it cann't download from website。

Comment (zxb789) [reply ADOPTERS.md]: sorry, the second question i don't know why has twice on landscape。it cloud be show up only one times.  thanks

**Comment by zxb789** [comment]
> would you modify the title of the pr to something like "add H3C to the list of ADOPTERS", and elaborate a bit more about what you are doing with KubeVirt?

ok  thanks,i will modify it

**Comment by dankenigsberg** [comment]
It would be a good idea to add a link to https://www.h3c.com/en/Products_Technology/Enterprise_Products/Cloud_Computing/Cloud_Computing_Products/H3C_CloudOS/H3C_CloudOS_full-stack/ (if this is indeed the product that uses KubeVirt)

**Comment by zxb789** [comment]
yeah ,i have modify it

**Comment by zxb789** [comment]
@dankenigsberg  hi，i have modified yesterday, but this pr can't reopen，plz review it at https://github.com/kubevirt/kubevirt/pull/4714. thanks

**Pull request closed without merging**